core/src/main/java/com/linecorp/armeria/server/HttpServerHandler.java

```diff
@@ -392,6 +392,7 @@ private void handleRequest(ChannelHandlerContext ctx, DecodedHttpRequest req) th
                     logBuilder.endRequest();
                 } else {
                     logBuilder.endRequest(cause);
+                    logBuilder.endResponse(cause);
```

Comment:
@trustin I found one more place that should add endResponse beside HttpRequestSubscriber#fail(cause) , could you confirm it?

Reply:
I think we don't need to call `endResponse(Throwable)` here because `HttpResponseSubscriber` will handle it. How about adding some comment instead?